config/initializers/active_admin.rb

```diff
@@ -169,19 +169,8 @@
   #
   # To load a javascript file:
   #   config.register_javascript 'my_javascript.js'
-  config.register_stylesheet 'admin/podemos.css'
   config.register_javascript 'admin/podemos.js'
```

Comment:
You mean the `podemos.js` file? Probably not currently, but it might be used in the administration console for functionality that we'll enable in the following phases. Didn't check into it.